### PR TITLE
Make publish history on an article optional.

### PR DESCRIPTION
**Pull request by nigelhanlon**
#### This resolves Clubhouse issue: [Make publishHistory optional in validator](https://app.clubhouse.io/condenastinternational/story/11811/make-publishhistory-optional-in-validator)

- [x] Tests have been written
- [x] All TODOs have a task created and assigned
- [x] Any tech debt raised as cards in Clubhouse
- [x] Checked that the new code runs properly
- [ ] Any schema changes have been added to the change log
- [x] Any schema changes are reflected in the Integration API docs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@condenast/flyway-schema-validator",
-  "version": "3.0.3",
+  "version": "3.0.4",
```
Comment (srilq) [inline package.json]: I think this is a breaking change so will require a major version change. (e.g. `flyway-service` currently assumes that `publishHistory` is always present and might throw a type error if it's missing).

Comment (nigelhanlon) [reply package.json]: Depending on how strictly we are following semantic versioning, should we bump to 4.0.0?

Comment (srilq) [reply package.json]: I think we will need to go to `4.0.0`, otherwise the service will break.

Comment (nigelhanlon) [reply package.json]: Actually I wonder should be wait to merge this until [Update Flyway service to migrate drafts if publishHistory omitted](https://app.clubhouse.io/condenastinternational/story/11813/update-flyway-service-to-migrate-drafts-if-publishhistory-omitted) is done?

Comment (srilq) [reply package.json]: It seems easier to me to just do a major version change now and then we can implement that in the service whenever we're ready.

Comment (nigelhanlon) [reply package.json]: Cool, I'm happy with that 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+2018-04-18, Version 4.0.0
+
+* Make `publishHistory` property optional to support draft migrations.
+
+2018-04-18, Version 3.0.3
+
+* Make `categories` a required field for articles and galleries.
```
Comment (srilq) [inline CHANGELOG.md]: For clarity, can we word this as a fix, e.g. ``Fix `categories` being an optional field on article and gallery.``

**Pull request merged**